compiler/src/dotty/tools/dotc/report.scala

```diff
@@ -134,6 +134,11 @@ object report:
   def enrichErrorMessage(errorMessage: String)(using Context): String = try {
     def formatExplain(pairs: List[(String, Any)]) = pairs.map((k, v) => f"$k%20s: $v").mkString("\n")
 
+    // Erasure typing runs with ctx.phase set to erasurePhase.next,
+    // so, if that's what ctx.phase is set to, we report the phase as "erasure".
+    // Doing so means that crashes in any of the mini phases right after erasure
+    // will also be reported as "erasure", but those are much less likely.
+    val phase    = if ctx.isRightAfterErasure then ctx.phase.prevMega else ctx.phase.currMega
```

Comment:
I'm not sure this is the best information to display: the stacktrace already tells us what code we're running, so I would prefer to just get the plain `ctx.phase` printed since that gives me extra information which is hard to extract from looking at the stacktrace.